search-enterprise/latest/admin/index.md

```diff
@@ -381,12 +381,17 @@ java -jar alfresco-elasticsearch-reindexing-4.0.0-app.jar \
   --alfresco.reindex.batchSize=100  \
```

Comment:
I'm not sure if you want to try to keep the version updated or not (it's probably an impossible job), but the latest 4.0.x version is 4.0.1.

Reply:
Let's keep it as current as we can.